cloud_governance/common/elasticsearch/elasticsearch_operations.py

```
@@ -141,13 +136,12 @@ def verify_elasticsearch_data_uploaded(self, index: str, uuid: str = '', workloa
         raise ElasticSearchDataNotUploaded
 
     @typechecked()
-    def upload_to_elasticsearch(self, index: str, data: dict, doc_type: str = '_doc', es_add_items: dict = None,
+    def upload_to_elasticsearch(self, index: str, data: dict, es_add_items: dict = None,
```

Ruff (0.15.14)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)